src/shared/ml-thirdparty-client/index.ts

```diff
@@ -209,10 +209,12 @@ export class Client {
    * Performs a request to generate a challenge for FIDO registration
    *
    * @param _consentId     identifier of consent as defined by Mojaloop API.
+   * @param destParticipantId   ID of destination - to be used when sending request
    */
   public async postGenerateChallengeForConsent(
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    _consentId: string
+    _consentId: string,
```

Comment:
what's with the `_` at the beginning of each variable name? as far as I understand, parameter variables shouldn't be named like that right as it is usually used for private class variables?

Reply:
that's just because they're not being used currently - once implemented we need to remove that

Reply:
Ohh I see, okay then :-)